nodejs/v3/nextjs-noappdir/app/src/pages/_app.tsx

```diff
@@ -1,6 +1,20 @@
-import '@/styles/globals.css'
-import type { AppProps } from 'next/app'
+import "@/styles/globals.css"
+import type { AppProps } from "next/app"
+import { appsignal } from "./_appsignal";
+import { ErrorBoundary } from "@appsignal/react";
+
+const FallbackComponent = () => (
+  <div>Uh oh! There was a client-side error :(</div>
+);
 
 export default function App({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return (
+  <ErrorBoundary
+    instance={appsignal}
+    tags={{ tag: "value" }}
+    fallback={(_error: Error) => <FallbackComponent />}
+  >
+      <Component {...pageProps} />
```

Comment:
Shouldn't this component throw an error in order to demonstrate that this works? (maybe I'm not seeing where that takes place?)

Reply:
It's here! https://github.com/appsignal/test-setups/pull/136/files#diff-38341284130b281218cfe59e0f1862f161cc7d2ecbf9e705e679edbf1b775612R11
In a child component.